### PR TITLE
Solved issue #313 - Button not visible properly

### LINKED ISSUE #313: Button not visible properly
When logged in to the application as an admin and while creating a new appointment, it is confusing to understand the add button. The visibility is poor due to color selection.

I am viewing this in chrome on mac. The button color is greenish.
screen shot 2016-02-17 at 11 13 43 pm

### PR DESCRIPTION
**Pull request by Anisha1234**
Fixes #313 .

**Changes proposed in this pull request:**
the disabled,active and hover buttons color states to a more understandable manner

 passes WCAG AA contrast accessibility test as #972 

1. The background color of Cancel Button in active state is changed from  $red_light to #08987f and font color is changes from $red to $white to give it a brighter view.

2. The background color of Cancel Button on hover state changes to background color rgba(255,109,111,0.75) i.e $red_lighter and font color changes to rgba(113,18,20) i.e $brown to help indicate what its function is.

3. The background color of Add Button in disabled state is changed from  $green to #dbebff and font color is changes from $white to $black to give it a better and lighter disable color.

4. The background color of Add Button in active state is changed from  $blue_light to #ff6d6f and font color is changes from $navy to $white to clearly signify the button activation.

- Previous state of button (In new appointment page)
   
1. Button Disabled state

olddisable

2. Button Active state

oldactive

3. Button On-hover state

oldhover

- Current state of button

   1.Button Disabled state

newdisabled

  2.Button Active state

newactive

3.Button On-hover state

newonhover

*Note: After this pull request the colours will make more sense*

cc @HospitalRun/core-maintainers

### CODE REVIEW

**Comment by jkleinsc** [comment]
@jglovier can you review this?  Does this help towards resolving #972?

File: app/styles/_variables_mixins.scss [hunk 1]
```diff
@@ -31,6 +32,7 @@ $sidebar_icon_lighter:  $navy_light;
 $sidebar_sublink_text:  #c2d2e3;
 $view_sub_nav:          #6784a2;
 $content_border:        $blue_lightest;
+$brown:                 #711214;
```
Comment (jglovier) [inline app/styles/_variables_mixins.scss]: I couldn't see in the screenshots where this color gets applied (and am unable to checkout the branch right now). Can you post a screencap of where this brown is used specifically please?

File: app/styles/components/_buttons.scss [hunk 2]
```diff
@@ -1,17 +1,17 @@
 .btn {
   &.on-white {
-    background-color: $blue_light;
-    color: $navy;
+    background-color: #08987f;
+    color: $white;
```
Comment (jglovier) [inline app/styles/components/_buttons.scss]: What do you think about swapping the hover styles with the values you've set on lines 3 and 4, so the lighter version is the default, and the hover version is the darker version? That would help keep the default state closer to the brand green.

File: app/styles/_variables_mixins.scss (hunk 1 above)
Comment (Anisha1234) [inline app/styles/_variables_mixins.scss]: screenshot

the $brown colour is used as font-colour of Cancel button on-hover state

File: app/styles/components/_buttons.scss (hunk 2 above)
Comment (Anisha1234) [inline app/styles/components/_buttons.scss]: I have swapped the hover style with active state style of Add button in the third commit.
- Active version is lighter version as default state
- Hover version is darker version
please check review the commit and suggest other invaluable changes

active state - 
acive

hover state -
hover

**Comment by Anisha1234** [comment]
Thank you so much @jglovier . I would really like to work establishing consistent button patterns/styleguide in the whole app regarding issue #972

**Comment by jkleinsc** [comment]
Looks good to me @Anisha1234.  I am going to merge this PR in.  Thanks for making HospitalRun better!

File: app/styles/_variables_mixins.scss (hunk 1 above)
Comment (jglovier) [inline app/styles/_variables_mixins.scss]: Thanks, but the screenshot I was looking for was of the UI showing the color in use. :-)

**Comment by jglovier** [comment]
@Anisha1234 thanks again for your work on this PR. :tada:

>I would really like to work establishing consistent button patterns/styleguide in the whole app regarding issue #972

Sounds great. Once I get an issue or PR going to specifically address that, I'd love your input there. :zap:

**Pull request merged**